### ISSUE TITLE
[BUG] UI doesn't show on BrailleNote Touch+

### ISSUE THREAD

**Issue by IsaMorphic** [SubverseIM/SubverseIM#555]
When running on Android Oreo (on the BrailleNote Touch+), after submitting the device lock PIN, the UI becomes unresponsive and shows a blank screen. Additionally, most UI elements are not readable by the screen reader. This is a regression when compared to previous versions of the app, which worked just fine on this device.

**Comment by IsaMorphic** [comment]
Still active on BrailleNote Touch+, despite working on the generic Android Oreo emulator.

**Comment by IsaMorphic** [comment]
Discovered that the source of this issue is in KeySoft itself. Reopening until a fix is confirmed.
